frontend/src/ui-components/AcmProgressTracker/AcmProgressTracker.tsx

```diff
@@ -112,6 +113,7 @@ export function AcmProgressTracker(props: AcmProgressTrackerProps) {
               </TextContent>
               {step.link && (
                 <AcmButton
+                  id={step.stepID && `${step.stepID}-link`}
```

Comment:
Added an id to the link button to help with testing, as multiple view logs can be rendered at a time.